docs/src/pages/[platform]/components/stepperfield/examples/StepperFieldThemeExample.tsx

```diff
@@ -0,0 +1,37 @@
+import { StepperField, ThemeProvider, Theme } from '@aws-amplify/ui-react';
```

Comment:
We may want to remove the `Theme` type? That's only working in Typescript?

Reply:
oh yes good catch the type has been removed